### PR TITLE
implemented web server

### PR DESCRIPTION
**Pull request by chinnaxs**
Implemented bare bones HTML web server with Go_Beer API Backend.
Supports:
listing of all beers
displaying beer details
delete beer
add beer
update beer

### CODE REVIEW

File: internal/pkg/web/handler/delete.go [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
+import (
+	"log"
+	"net/http"
+
+	"github.com/chinnaxs/go_beer_client/internal/pkg/api"
+)
+
+var deleteHandler *Handler
+
+func MakeDeleteBeerHandler(templatePath string, apiClient *api.ApiClient) http.HandlerFunc {
```
Comment (fehlmach) [inline internal/pkg/web/handler/delete.go]: Just call it DeleteBeerHandler

File: internal/pkg/web/handler/delete.go [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+	deleteHandler := Handler{
+		Templates: nil,
+		ApiClient: apiClient,
+	}
+	return deleteHandler.deleteBeerHandler
+}
+
+func (h *Handler) deleteBeerHandler(w http.ResponseWriter, r *http.Request) {
+	log.Printf("%s: %s", r.Method, r.URL.Path)
+	switch r.Method {
```
Comment (fehlmach) [inline internal/pkg/web/handler/delete.go]: If you have just one condition, use an if stmt

File: internal/pkg/web/handler/delete.go [hunk 3]
```diff
@@ -1,13 +1,38 @@
...
+	case http.MethodPost:
+		h.postDeleteBeerHandler(w, r)
+	default:
+		w.WriteHeader(http.StatusNotFound)
+	}
+}
+
+func (h *Handler) postDeleteBeerHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (fehlmach) [inline internal/pkg/web/handler/delete.go]: We should take about naming.

File: internal/pkg/web/handler/delete.go [hunk 4]
```diff
@@ -1,13 +1,38 @@
...
+	"net/http"
+
+	"github.com/chinnaxs/go_beer_client/internal/pkg/api"
+)
+
+var deleteHandler *Handler
+
+func MakeDeleteBeerHandler(templatePath string, apiClient *api.ApiClient) http.HandlerFunc {
+	deleteHandler := Handler{
+		Templates: nil,
```
[the remark this answers is not in the document]
Comment (chinnaxs) [reply internal/pkg/web/handler/delete.go]: resolved

File: internal/pkg/web/handler/delete.go (hunk 1 above)
Comment (chinnaxs) [reply internal/pkg/web/handler/delete.go]: resolved

File: internal/pkg/web/handler/delete.go (hunk 3 above)
Comment (chinnaxs) [reply internal/pkg/web/handler/delete.go:28]: implemented more consistent naming throughout all handlers

**Pull request merged**